src/templates/init/test/sample-unit-test.spec.ts

```diff
@@ -0,0 +1,5 @@
+describe('Sample unit test', () => {
+  it('should do some unit test',  () => {
+    expect(2+2).toBe(4);
```

Comment:
I would add an actual unit test. A good candidate would be something like `normalizePort` from `./bin/www`.